includes/providers/gam/class-gam-api.php

```diff
@@ -5,6 +5,8 @@
  * @package Newspack
```

Comment:
There are three instances of the legacy `Newspack_Ads_Model` class name being called in this file, on lines 1438, 1448, and 1452. These cause a fatal error to occur when connecting to GAM for the first time, which results in an incomplete configuration.

Reply:
Thank you for catching that! Fixed by 538ff78060837eb40ef7cb3f14a0fc8fac25c573

Reply:
Confirmed, thanks!